doc/code-style.markdown

```diff
@@ -8,14 +8,6 @@ Guidelines that could be easily automated, have been. Please install:
 - [EditorConfig plugin][editorconfig], so your editor can apply some of the
   formatting right as you type;
 - `clang-format` (part of `clang` package in most Linux distributions);
```

Comment:
Please remove that line as well, as there's no requirement to run `clang-format` now (for the same reason as we disabled the pre-commit hook).